packages/web3-providers/src/rarible/index.ts

```diff
@@ -51,21 +51,22 @@ function createERC721TokenFromAsset(
     tokenId: string,
     asset?: RaribleNFTItemMapResponse,
 ): ERC721TokenDetailed {
-    const imageURL = resolveResourceLink(asset?.meta.image?.url.ORIGINAL ?? asset?.meta.image?.url.PREVIEW ?? '')
+    const imageURL = resolveResourceLink(asset?.meta?.image?.url.ORIGINAL ?? asset?.meta?.image?.url.PREVIEW ?? '')
```

Comment:
Access `assets.meta` so many times!

```suggestion
    const meta = assets?.meta || {}
    const imageURL = resolveResourceLink(meta.image?.url.ORIGINAL ?? metaimage?.url.PREVIEW ?? '')
```